components/StartButton.js

```diff
@@ -0,0 +1,38 @@
+import { View, Text, TouchableOpacity } from "react-native";
+
+export const StartButton = ({ isRunning, startGame }) => {
+  if (isRunning) return null;
```

Comment:
we can provide a case to prevent all of the code below being run by just returning null inside the component, increasing performance